CHANGELOG.md

```diff
@@ -26,6 +29,10 @@
 
 ##### Enhancements
 
+* Add a post_integrate_hook API  
+  [lucasmpaim](https://github.com/lucasmpaim)
+  [#7432](https://github.com/CocoaPods/CocoaPods/issues/7432)
```

Comment:
looks like this was added twice - possibly due to a rebase

Reply:
Yeah, in last rebase this are duplicated, I have fixed this, Thanks.